duckduckgo/macos-browser
# Privacy icon animation with tracker entities

Task/Issue URL: https://app.asana.com/0/0/1200885599483230/f

**Description**:
_Note: Review will be a bit hard because of too many changes in Assets. Sorry about that. 🙏_ 

The PR contains the next iteration of changes in privacy icon animation. PNG sequences were replaced with Lottie animation files. Tracker animation contains starting letters of tracking entities.

**Steps to test this PR**:
1. Visit multiple websites and compare information from tracker animation with dashboard info.
2. Make sure unencrypted HTTP connection contains privacy shield icon with red dot (www.dsl.sk)

**Testing checklist**:

_Test with Release configuration is not necessary_

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
